### ISSUE TITLE
Calls to EditorView.update are not allowed while an update is in progress

### ISSUE THREAD

**Issue by mbui0327** [uiwjs/react-codemirror#374]
I ran into this error after update to cm6. Anyone can shed some light here. Thanks.
```
index.js?4948:6173 Uncaught Error: Calls to EditorView.update are not allowed while an update is in progress
    at EditorView.update (index.js?4948:6173:1)
    at EditorView._dispatch (index.js?4948:6092:1)
    at EditorView.dispatch (index.js?4948:6160:1)
    at eval (useCodeMirror.js?fd7b:143:1)
    at invokePassiveEffectCreate (react-dom.development.js?61bb:23487:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js?61bb:3945:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?61bb:3994:1)
    at invokeGuardedCallback (react-dom.development.js?61bb:4056:1)
    at flushPassiveEffectsImpl (react-dom.development.js?61bb:23574:1)
    at unstable_runWithPriority (scheduler.development.js?3069:468:1)
    at runWithPriority$1 (react-dom.development.js?61bb:11276:1)
    at flushPassiveEffects (react-dom.development.js?61bb:23447:1)
    at flushDiscreteUpdates (react-dom.development.js?61bb:22356:1)
    at flushDiscreteUpdatesIfNeeded (react-dom.development.js?61bb:3768:1)
    at dispatchDiscreteEvent (react-dom.development.js?61bb:5886:1)
    at eval (index.js?4948:2614:1)
    at DOMObserver.ignore (index.js?4948:5632:1)
    at DocView.updateSelection (index.js?4948:2592:1)
    at EditorView.update (index.js?4948:6215:1)
    at DOMObserver.flush (index.js?4948:5736:1)
    at DOMObserver.forceFlush (index.js?4948:5692:1)
    at EditorView.measure (index.js?4948:6304:1)
    at eval (index.js?4948:6453:1)
```

**Comment by jaywcjlove** [comment]
Give me an example to reproduce the error?
@mbui0327

**Comment by mbui0327** [comment]
Oh my fault. I have fixed this by upgrade to react 18.0.2. The previous version was 17.*.
